playbooks/setup.yml

```diff
@@ -9,7 +9,8 @@
 - name: python check
   hosts: ofn_servers
   gather_facts: no
-  remote_user: root
+  # Changed while using AWS services
+  remote_user: ubuntu
```

Comment:
this is not going to work for all servers, right?

Reply:
Yeah, we can't merge this change either. There's a comment here with a workaround for AWS servers: https://github.com/openfoodfoundation/ofn-install/blob/58f31e7cf0a68ae70bd0faebe6df9274ef1ec730/playbooks/setup.yml#L4-L7